python_modules/dagster-test/dagster_test/toys/partitioned_assets/with_backfill_policies.py

```diff
@@ -0,0 +1,19 @@
+from dagster import BackfillPolicy, DailyPartitionsDefinition, MaterializeResult, asset
+
+
+@asset(
+    partitions_def=DailyPartitionsDefinition(start_date="2023-01-01"),
+    backfill_policy=BackfillPolicy.single_run(),
+    group_name="with_backfill_policy",
+)
+def successful_daily_asset() -> MaterializeResult:
```

Comment:
maybe call this `single_run_backfill_policy` or `single_run_backfill_policy`? for someone browsing assets in go/toys, that will make it clear what this one is there for.

Reply:
updated!